gatsby/content/projects/2015-11-29-ruma.mdx

```diff
@@ -1,19 +0,0 @@
----
-layout: project
-title: Ruma
-categories:
-  - server
-description: Ruma is a server written in Rust.
-author: Jimmy Cuadra
-maturity: Not actively maintained
-language: Rust
-license: MIT
-repo: https://github.com/ruma/ruma
-home: https://www.ruma.io/
-room: "#ruma:matrix.org"
----
-
-The Ruma Rust-language homeserver project is [currently not maintained](https://github.com/ruma/ruma), but the *Ruma project*,
-including [ruma-api](https://github.com/ruma/ruma-api]) (framework for building Matrix APIs in Rust) and
-[ruma-client-api](https://github.com/ruma/ruma-client-api) (request and response types for the Matrix client API)
-are still being actively worked on.
```

Comment:
Let's also remove it from https://github.com/matrix-org/matrix.org/blob/b93d07ab61ad670aed1fcbb6ea3348a424524d48/gatsby/src/pages/faq.js#L1389

Reply:
Yeah, that should be removed / replaced with a link to [Conduit](https://conduit.rs/).